dlt/common/storages/fsspecs/google_drive.py

```diff
@@ -367,17 +162,30 @@ def makedirs(self, path: str, exist_ok: Optional[bool] = True) -> None:
                 return
             else:
                 raise FileExistsError(path)
-        if self._parent(path):
+        if len(path.split("/")) > 2:
             self.makedirs(self._parent(path), exist_ok=True)
         self.mkdir(path, create_parents=False)
 
+    def _is_path_root_id(self, path: str) -> bool:
+        """Checks if path is root id"""
+        return len(path.split("/")) == 0
```

Comment:
I don't think that's a correct statement. `split()` returns a list with at least one element - with no separator, it'll still have `len() == 1`.

![image](https://github.com/dlt-hub/dlt/assets/26030071/eda3eaa1-81dd-4d04-b9a2-966239f1f7ed)

Reply:
you are so right. that needs to be fixed